src/platform/forms-system/src/js/components/ConfirmationView/ChapterSectionCollection.jsx

```diff
@@ -116,7 +128,11 @@ const fieldEntries = (key, uiSchema, data, schema, schemaFromState, index) => {
   } else if (uiSchema['ui:options']?.labels?.[refinedData]) {
     refinedData = uiSchema['ui:options'].labels[refinedData];
   } else if (
-    uiSchema['ui:webComponentField']?.identifier === 'VaCheckboxGroupField'
+    uiSchema['ui:webComponentField']?.identifier === 'VaCheckboxGroupField' ||
+    (uiSchema?.['ui:field']?.WrappedComponent.name === 'FileField' &&
```

Comment:
Oh, I didn't realize where this WrappedComponent was coming from. Looks like redux/connect. So if they use connect, then WrappedComponent will be present. This seems ok to me 👍